### DISCUSSION TITLE
What is the relationship between Incremental Static Regeneration and BandWidth?

### DISCUSSION THREAD

**Discussion by matamatanot** [vercel/vercel#5308] [Help]
We've been working on a new product development using Next.js + Vercel for a while now. It's been a great development experience and the product has taken shape quickly.

We have a few concerns before the release. **Pro or Enterprise?**

We are currently paying $230 per month for 4 members + PassWord Protection. Bandwidths are only about 100MB a month.

We don't use the Serverless Function. However, we are planning to use Incremental Static Regeneration.
https://vercel.com/docs/platform/limits#general-limits

What is the relationship between Incremental Static Regeneration and BandWidth?
Considering that Deployments Created per Day is 3000, how do I set the revalidate time?

**Comment by paulogdm** [comment]
> What is the relationship between Incremental Static Regeneration and BandWidth?

Can you elaborate? You can count as bandwidth all traffic from a Vercel edge to your customer.

> Considering that Deployments Created per Day is 3000, how do I set the revalidate time?

I need you to elaborate here. iSG does not relate to "Deployments Created per Day" or any other rate-limits.

**Reply by matamatanot** [reply]
> iSG does not relate to "Deployments Created per Day or rate-limits.

Is that true? 
We will use getStaticProps and we are considering setting the revalidate time to relatively short. If a lot of pages(paths) are created, It looks like we will soon have over 3000 builds running. So these are not included in **the Deployments Created per Day 3000**?

Could it be that iSG has something to do with  "**Serverless Function Execution**" and "**Build Execution**" ?
https://vercel.com/docs/platform/fair-use-policy?query=bandwd#typical-monthly-usage-guidelines
